net_printer/benchmarks.md

```diff
@@ -0,0 +1,27 @@
+# Benchmarks
+
+This page describes the suite of benchmark designs that are used to assess
+routing performance. The full list of benchmark designs, along with links to
+the original sources and some basic statistics is provided in the following
+table:
+
+|Source Benchmark Suite|Benchmark Name|LUTs|FFs|DSPs|BRAMs|
+|----------------------|--------------|----|---|----|-----|
+| [VTR](https://docs.verilogtorouting.org/en/latest/vtr/benchmarks/#vtr-benchmarks)|`mcml`|43k|15k|105|142|
+| [Rosetta](https://github.com/cornell-zhang/rosetta)|`fd` (face-detection)|46k|39k|72|62|
```

Comment:
Maybe a footnote saying this is an HLS design and compiled using Vitis HLS?

Reply:
perhaps two additional columns: "Distributed As" and "Formatting Steps"?:
* vtr_mcml: Verilog, Vivado Synth.
* rosetta_fd: C++, Vitis HLS Synth.
* koios_dla_like_large: Verilog, Vivado Synth.
* ispd16_example2: Vivado Design Checkpoint, device re-targeted with Vivado
* boom_soc: code generation frame work, generate instance, Vivado Synth

Reply:
That's a lot of information to put in a table. Let's leave it -- the purpose of this page is not to let people reproduce the placed benchmarks exactly.